### PR TITLE
added missing pieces for MPM new protocols

### PR DESCRIPTION
**Pull request by raphaelcoeffic**
related to #6640

### CODE REVIEW

**Comment by 3djc** [comment]
True, suggesting minor string changes too (don't forget to prefix your branch !)

**Comment by pascallanger** [comment]
You don't need it since there is a catch all at the end:
```
  // Sentinel and default for protocols not listed above (MM_RF_CUSTOM is 0xff)
  {0xfe,                            0, false,      NO_SUBTYPE,            nullptr}
```
You only need to add a line if there is a subtype or option is needed.
Pascal

**Comment by 3djc** [comment]
yes,  but if you add more later, it may come back and bite you :)

**Comment by pascallanger** [comment]
Then you have some work to do to define all of them... None of the protocols which does not have subtype or option are defined.

**Comment by pascallanger** [comment]
So for me you do them all or none but not half...

**Comment by 3djc** [comment]
You have a very valid point Pascal

**Comment by raphaelcoeffic** [comment]
Right, seems the changes in the table are not needed: it’s using a search based on protocol ID, so no problem here. Let’s remove those without options.

**Comment by 3djc** [comment]
So we can close this and i'll  cherry pick the translation screen

**Comment by raphaelcoeffic** [comment]
Companion changes are still needed ;-)

**Comment by MRC3742** [comment]
I have all the changes done for Companion and will submit a PR later today.
Also I have updates for 2.2.4 if you still want to update the 2.2 branch?

Rich

**Comment by bsongis** [comment]
@MRC3742 We have said there we wouldn't release any new 2.2 version. I hope we are not lying, it's difficult to know in advance...

File: radio/src/translations/untranslated.h [hunk 1]
```diff
@@ -129,5 +129,5 @@
 #define LEN_DSM_PROTOCOLS              "\004"
 #define TR_DSM_PROTOCOLS               "LP45""DSM2""DSMX"
 
-#define LEN_MULTI_PROTOCOLS            "\006"
-#define TR_MULTI_PROTOCOLS             "FlySky""Hubsan""FrSky\0""Hisky\0""V2x2\0 ""DSM\0  ""Devo\0 ""YD717\0""KN\0   ""SymaX\0""SLT\0  ""CX10\0 ""CG023\0""Bayang""ESky\0 ""MT99XX""MJXq\0 ""Shenqi""FY326\0""SFHSS\0""J6 PRO""FQ777\0""Assan\0""Hontai""OpnLrs""FS 2A\0""Q2x2\0 ""Walk.\0""Q303\0 ""GW008\0""DM002\0""Cabell""Esy150""H8 3D\0""Corona""CFlie\0""Hitec\0""WFly\0 ""Bugs\0 ""BugMin""Traxas""NC1701""E01X\0 ""V911S\0""GD00X\0""V761\0 ""KF606\0""Redpin""Potens""ZSX\0  ""FLYZON"
+#define LEN_MULTI_PROTOCOLS            "\007"
```
Comment (MRC3742) [inline radio/src/translations/untranslated.h:132]: I'm not sure you want to expand the Protocol to seven characters as the space in the X9D+ screen is limited to six? Not sure on the other Radios if this is also limited? Please review before changing.
Thanks - Rich

**Comment by MRC3742** [comment]
@bsongis  Okay on 2.2.4
Are the nightlies still being built in this branch?
I don't mind updating it for the benefit of someone using this option.

File: radio/src/translations/untranslated.h (hunk 1 above)
Comment (bsongis) [reply radio/src/translations/untranslated.h:132]: I see this comment too late. It needs to be checked on 128x64 board, perhaps I pressed the button too quickly

**Comment by bsongis** [comment]
I don't think we have 2.2 nightlies today

**Comment by 3djc** [comment]
We do if changes are pushed in 2.2 branch

**Comment by bsongis** [comment]
If we keep nightlies alive, it means we know we are lying :) 

Either we want to release a new 2.2 (and we keep nightlies), either we stop nightlies (and pull requests). 

Difficult choice. I would vote for keeping 2.2 open, with nightlies and pull requests. And we just say that if 2.3 is ready soon, perhaps there will be no new 2.2 release

**Comment by bsongis** [comment]
ok. we keep 2.2 open. but only for critical issues, so definitely not this one :)

**Pull request merged**